maat-court-data-api/src/test/java/gov/uk/courtdata/builder/TestEntityDataBuilder.java

```diff
@@ -593,4 +593,17 @@ public static RepOrderCCOutComeEntity getRepOrderCCOutcomeEntity(Integer repOder
                 .id(repOderOutComeId)
                 .build();
     }
+
+    public static CorrespondenceEntity getCorrespondenceEntity(Integer id) {
+
+        return CorrespondenceEntity.builder()
+                .repId(REP_ID)
+                .id(id)
+                .generateDate(LocalDateTime.now())
```

Comment:
We can reduce some of the duplication here by introducing a local variable, this will also ensure that the time for all 3 fields is truly identical (down to the nearest ms)

`LocalDateTime now = LocalDateTime.now();`

Reply:
Insert current time to h2 for testing. i think this should fine